src/components-examples/material/chips/chips-reactive-form/chips-reactive-form-example.ts

```diff
@@ -0,0 +1,94 @@
+import {COMMA, ENTER} from '@angular/cdk/keycodes';
+import {Component, inject} from '@angular/core';
+import {MatChipEditedEvent, MatChipInputEvent, MatChipsModule} from '@angular/material/chips';
+import {MatIconModule} from '@angular/material/icon';
+import {JsonPipe, NgFor} from '@angular/common';
+import {MatFormFieldModule} from '@angular/material/form-field';
+import {LiveAnnouncer} from '@angular/cdk/a11y';
+import {FormsModule, ReactiveFormsModule, FormControl, FormGroup, Validators} from '@angular/forms';
+
+export interface Fruit {
+  name: string;
+}
+
+/**
+ * @title Chips with reactive forms
+ */
+@Component({
+  selector: 'chips-reactive-form-example',
+  templateUrl: 'chips-reactive-form-example.html',
+  styleUrls: ['chips-reactive-form-example.css'],
+  standalone: true,
+  imports: [
+    MatFormFieldModule,
+    MatChipsModule,
+    NgFor,
+    MatIconModule,
+    FormsModule,
+    ReactiveFormsModule,
+    JsonPipe,
+  ],
+})
+export class ChipsReactiveFormExample {
+  addOnBlur = true;
+  readonly separatorKeysCodes = [ENTER, COMMA] as const;
+
+  form: FormGroup = new FormGroup({
+    fruits: new FormControl<Fruit[]>(
+      [{name: 'Lemon'}, {name: 'Lime'}, {name: 'Apple'}],
+      Validators.required,
+    ),
+  });
+
+  get fruits() {
+    return this.form.get('fruits');
+  }
+
+  announcer = inject(LiveAnnouncer);
+
+  constructor() {}
+
+  add(event: MatChipInputEvent): void {
+    // Check if input is null
+    if (this.fruits == null) return;
+
+    const value = (event.value || '').trim();
+    // Add our fruit
+    if ((value || '').trim()) {
```

Comment:
The `(value || '').trim()` here isn't necessary since it was done in the const above already.